withdrawn-packages.txt

```diff
@@ -6,3 +6,6 @@ redis-sentinel-7.2-bitnami-compat-7.4.0-r0.apk
 redis-cluster-7.2-bitnami-compat-7.4.0-r0.apk
 rancher-kontainer-driver-metadata-2.9-r0.apk
 rancher-loglevel-0.1.6-r0.apk
+thin-provisioning-tools-1.0.13-r1.apk
```

Comment:
We normally don't remove any old packages after withdraws - I think we can drop these,  unless theres another reason?

Reply:
I remove the change to `withdrawn-packages.txt` which will then leave thin-provisioning-tools in the public wolfi archive, if that is the standard method here.

However, doing so means that in the future when there is a vulnerability against thin-provisioning-tools, a user of `apk add thin-provisioning-tools` will blissfully go on with life believing that they have up to date software.